examples/large_table/static/index.html

```diff
@@ -3,7 +3,6 @@
     <head>
         <meta charset="utf-8">
         <title>Table100x100 Test</title>
-        <link rel="stylesheet" href="styles.css">
```

Comment:
I get why you had to do this but it's not a great side effect. I wouldn't want people thinking that including strings inline is the right way to do things. I'd much rather examples use this approach (`<link>` in `<head>`). How about we keep this and instead of including style strings in each individual example, we include them all in the showcase example project?

Reply:
Updated to load styles in at the showcase level.